aviary/utils/process_input_decks.py

```diff
@@ -139,7 +139,7 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
                 initial_guesses[var_name] = float(var_values[0])
                 continue
 
-            if aircraft_values.get_val('verbosity').value >= 2:
+            if aircraft_values.get_val('settings:verbosity').value >= 2:
```

Comment:
Can you use `Settings.VERBOSITY` for the three `get_val`s here as well?

Reply:
Done.